azext_iot/tests/central/test_iot_central_int.py

```diff
@@ -25,10 +25,7 @@
 DEVICE_ID = os.environ.get("azext_iot_central_device_id")
 TOKEN = os.environ.get("azext_iot_central_token")
 DNS_SUFFIX = os.environ.get("azext_iot_central_dns_suffix")
-
```

Comment:
If both v1 and preview are officially supported, then ideally the IT's can do a pass against each supported API version. If we only do a pass against v1 we have much less confidence for preview at any given release.

Reply:
Wont-fix: preview is expected to have breaking changes and will result in frequent updates.